lib/config/default.js

```diff
@@ -934,5 +931,49 @@ exports.config = () => ({
      * loop data.
      */
     native_metrics: { enabled: true }
+  },
+
+  /**
+   *
+   * Controls the behavior of Logs in Context within agent
+   * TODO: sorting out some default values as the spec differs from conversations
+   */
+  application_logging: {
+    /**
+     * Toggles all application logging features, regardless of more specific flagging.
+     *
+     * @env NEW_RELIC_APPLICATION_LOGGING_ENABLED
+     */
+    enabled: false,
```

Comment:
not sure i follow what you mean

Reply:
well there's a setting to turn off log enriching and this master switch. from the agent we could turn off the auto-injection but seems like we'll potentially need/want to plumb for turning off for either setting when the enrichers are manually applied by customers.

Reply:
Looking at the Winston auto-instrumentation, there will likely be a layer in-between our instrumentation (in the log enrichers repo) and our module-loading hooks in the agent, so that the former will probably not need to be aware of these flags while the latter certainly will need to be. Does that track?